CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# Contributing
+
+Thank you for your interest in improving InsightConnect! Please review our [Code of Conduct] before making contributions.
```

Comment:
Thank you for your interest in joining the InsightConnect Developer community!

Reply:
Nice!

Reply:
Updated